src/eredis.app.src

```diff
@@ -4,7 +4,6 @@
               {modules,[eredis,eredis_client,eredis_parser,eredis_sub,
                         eredis_sub_client]},
               {registered,[]},
-              {applications,[kernel,stdlib,ssl,public_key,certifi,parse_trans,
-                             ssl_verify_fun]},
+              {applications,[kernel,stdlib,ssl,tls_certificate_check]},
```

Comment:
What happened with `public_key`? Was it not valid before?

Reply:
`eredis` will no longer using it after this PR.